lib/rubocop/config_validator.rb

```diff
@@ -104,13 +104,17 @@ def check_target_ruby
     def alert_about_unrecognized_cops(invalid_cop_names)
       unknown_cops = list_unknown_cops(invalid_cop_names)
 
+      return unless unknown_cops.any?
```

Comment:
Is it something like this? It may be a little simpler In this case.

```suggestion
      return if unknown_cops.empty?
```

Reply:
Thank you, I've updated.